ambari-logsearch/ambari-logsearch-web/src/app/components/login-form/login-form.component.html

```diff
@@ -16,7 +16,7 @@
 -->
 
 <div class="login-form well col-md-4 col-md-offset-4 col-sm-offset-4">
-  <div class="alert alert-danger" *ngIf="isLoginAlertDisplayed" [innerHTML]="'authorization.error' | translate"></div>
+  <div class="alert alert-danger" *ngIf="isLoginAlertDisplayed" [innerHTML]="errorMessage"></div>
```

Comment:
No need to use `innerHTML` attribute since there are no more HTML tags in the message